packages/orchestrator-ui-components/src/components/WfoSettingsModal/WfoSubmitModal.tsx

```diff
@@ -18,6 +18,7 @@ export type WfoSubmitModalProps = {
     onClose: () => void;
     onSubmit: () => void;
     isLoading: boolean;
+    isDisabled?: boolean;
```

Comment:
Looking at the usage of this prop it seems you only prevent the submit part. You can still cancel. Suggesting to rename this prop to `isSubmitDisabled` or something that explains what is being disabled